VectorRumble.Shared/VectorRumble.Shared.projitems

```diff
@@ -66,4 +66,26 @@
   <ItemGroup>
     <Folder Include="$(MSBuildThisFileDirectory)Content\" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)\Strings.de.resx">
+      <Link>Strings.de.resx</Link>
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Strings.de.Designer.cs</LastGenOutput>

```

Comment:
@CartBlanche do we still need this MSBuild Metadata? Same for `Generator`?

Reply:
Does it automagically know how to generate the right file for each resx? I'll double check and test it later.

Reply:
Btw, I think we do, because under UWP, it would be looking for a `resw` and we want it to share a `resx`. But again I'll test it later.

Reply:
does doesn't need to generate the Strings.de.Designer.cs files at all. We just need the `Strings.de.resx` to be an embedded resource. Same for the Spanish. So these items to no need the Generator or LastGenOutput metadata items.